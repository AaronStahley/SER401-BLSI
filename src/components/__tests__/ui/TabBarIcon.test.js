import "react-native";
import React from "react";
import TabBarIcon from "../../ui/TabBarIcon";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<TabBarIcon/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correctly", () => {
    const tree = renderer.create( <TabBarIcon name="md-thumbs-up"/> ).toJSON();
    expect(tree).toMatchSnapshot();
});

it("handles false focused correctly", () => {
  const tree = renderer
    .create(<TabBarIcon name="md-thumbs-up" focused={false} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("handles true focused correctly", () => {
  const tree = renderer
    .create(<TabBarIcon name="md-thumbs-up" focused={true} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
