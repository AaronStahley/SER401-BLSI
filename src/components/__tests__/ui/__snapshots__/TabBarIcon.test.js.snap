// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handles false focused correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;

exports[`handles true focused correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;

exports[`renders correctly 2`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;
